sbin/db_updater.py

```diff
@@ -73,7 +73,10 @@ def nbelement(collection=None):
 def dropcollection(collection=None):
     if collection is None:
         return False
-    return db.dropCollection(collection)
+    if collection == "cve":
+        collection = "cves"
+    ret = db.dropCollection(collection)
+    return ret
```

Comment:
This should be changed to "cves" right? The sources has "cve" as name, but in the database the table is called "cves".

Reply:
Indeed the collection is `cves`.  It sounds like an old bug ;-)